src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -1243,6 +1221,88 @@ lExit: ;
 }
 FCIMPLEND
 
+static FC_BOOL_RET IsFastPathSupportedHelper(FieldDesc* pFieldDesc)
```

Comment:
```suggestion
static bool IsFastPathSupportedHelper(FieldDesc* pFieldDesc)
```
`FC_BOOL_RET` is only meant to be used in FCALL declarations.

Reply:
Sure; I was trying to use the same return value as the FCALL caller to avoid the TRUE : FALSE conversion, but I'll change it to `bool`.